### DISCUSSION TITLE
Exclude or delete thumbnails

### DISCUSSION THREAD

**Discussion by Liro67** [ivandokov/phockup#125] [Ideas]
Thanks for the great tool, love it! I was sorting 76k+ files from almost 20 years of pictures with it. If I can suggest an improvement, would be good to be able to delete all kind of thumbnails files like the windows Thumbs.db or the Apple *.ithm or *.ithmb. Thanks.

**Comment by ivandokov** [comment]
Hello @Liro67 and that's for the suggestion. 
I need some clarification - by "delete all kind of thumbnails" do you mean simply not move them to the output directory?

**Comment by Liro67** [comment]
Well yes but I would even like to have a parameter to choose to simply delete them. My personal experience with sorting almost 20 years of pictures is that these files where created by a device and then backup by the software or copy function used at that time. Now I want a clear, simple and useable database where those files are completely irrelevant! 🙂
Many thanks for sharing these great tool!

**Comment by ivandokov** [comment]
To be honest I do not think this is a feature that should be build in the software or at least it is not a significat one since the same result can be accomplished by a `rm` command after the work is done. Also these files are there for a reason and if we do want to include this option in it should be either with an opt-in or opt-out flag.

**Comment by pabera** [comment]
Isn't this request targeting [this line](https://github.com/ivandokov/phockup/blob/master/src/phockup.py#L15) to be more flexible? E.g. by adding allowing the user to ignore specific files/folders through an argument? I have the same use case on my NAS where I want to ignore (but not delete!) automatically generated thumbnails. I am currently solving it [this way](https://github.com/pabera/phockup/blob/more-ignored-files/src/phockup.py#L15).
